src/shared/cgroup-setup.c

```diff
@@ -17,6 +17,67 @@
 #include "fileio.h"
 #include "user-util.h"
 #include "fd-util.h"
+#include "virt.h"
```

Comment:
the headers are currently not sorted. if you touch that you might as well sort it properly

Reply:
will do